README.md

```diff
@@ -66,3 +66,189 @@ No methods are available for this component
 | Name | Description |
 | ---  | ---         |
 | timeZoneUpdated | Emitted when timeZone is changed |
+
+
+# Proposed Components
+
+## DxpForm
+
+This component handles creation and validation of the forms in the app. It hides the logic for form creation and validation from the developers and they can directly define the form elements and pass the schema in this component that renders a Ionic form to be used in the app.
+
+## Usage
+
+```html
+<template>
+  <DxpForm :schema="{}" :fabSaveButton="true" :blockSaveButton="false" />
```

Comment:
I think the save button should only have one option, use the default or provide a custom button. field is something like:

saveButtonId: ""

In this the user passes the element id of their save button where the action is attached. Internally the component should go to that button and add the form id to that button during code compile

Reply:
@dt2patel Sir, as discussed, I am exploring on this and it seems like fab-button does not work like a normal button with form.

An ion-fab-button cannot be directly bound to a form for submission in the same way a standard ion-button with type="submit" can. This is because the ion-fab-button component lacks the internal logic to handle form submissions, which is present in ion-button.